meta-lmp-base/recipes-security/optee/optee-os-tadevkit_3.17.0.bb

```diff
@@ -0,0 +1,25 @@
+# Compatible with optee-os-fio and optee-os from meta-arm
+require ${@bb.utils.contains('PREFERRED_PROVIDER_virtual/optee-os', 'optee-os', 'recipes-security/optee/optee-os_${PV}.bb', '', d)}
+require ${@bb.utils.contains('PREFERRED_PROVIDER_virtual/optee-os', 'optee-os-fio', 'optee-os-fio_${PV}.bb', '', d)}
+require ${@bb.utils.contains('PREFERRED_PROVIDER_virtual/optee-os', 'optee-os-fio-mfgtool', 'optee-os-fio-mfgtool_${PV}.bb', '', d)}
+
+SUMMARY = "OP-TEE Trusted OS TA devkit"
+DESCRIPTION = "OP-TEE TA devkit for build TAs"
+HOMEPAGE = "https://www.op-tee.org/"
+
+# Needed due provides from optee-os-fio (for virtual/optee-os)
+PROVIDES = "${PN}"
+
+do_install() {
+    #install TA devkit
+    install -d ${D}${includedir}/optee/export-user_ta/
+    for f in ${B}/export-ta_${OPTEE_ARCH}/* ; do
+        cp -aR $f ${D}${includedir}/optee/export-user_ta/
+    done
+}
+
+do_deploy() {
+    echo "Do not inherit do_deploy from optee-os."
```

Comment:
`bbplain` or `bbnote` can do the same job of `echo` and is more bitbake friendly
```
# Print the output exactly as it is passed in. Typically used for output of
# tasks that should be seen on the console. Use sparingly.
# Output: logs console
bbplain() {

# Notify the user of a noteworthy condition. 
# Output: logs
bbnote() {
```
https://git.yoctoproject.org/poky/plain/meta/classes/logging.bbclass

Reply:
Yeah, this was basically a copy from https://github.com/jonmason/meta-arm/blob/master/meta-arm/recipes-security/optee/optee-os-tadevkit_3.16.0.bb (with a rev bump), so kept the same message.

But I think here this message was supposed to be a comment instead,  since there is no need to notify the user that this function is a no-op. Will update anyway.

Once we get a working main we will start doing the rev bumps in meta-arm directly.

Reply:
If it is a copy I think it's better to have it as it is and submit a patch to meta-arm but this is mainly a cosmetic change and we don't need to waste much time on it

Reply:
Yeah, that is my thinking as well, prefer to keep with the same "issues" and fix upstream in meta-arm.